src/Common/ErrorCodes.cpp

```diff
@@ -590,6 +590,7 @@
     M(708, ILLEGAL_STATISTIC) \
     M(709, CANNOT_GET_REPLICATED_DATABASE_SNAPSHOT) \
     \
+    M(998, FORBID_FOR_STREAMING_QUERIES) \
```

Comment:
I'm not sure what the exact convention is here, but it sure looks like this error belongs one line higher (with value 710), next to all the normal errors. The errors below correspond to big basic components.